networking/ovn_kubernetes_network_provider/configuring-hybrid-networking.adoc

```diff
@@ -14,7 +14,7 @@ Complete any further installation configurations, and then create your cluster.
 [id="configuring-hybrid-networking-additional-resources"]
 == Additional resources
 
-* xref:../../windows_containers/understanding-windows-container-workloads.adoc#understanding-windows-container-workloads[Understanding Windows container workloads]
-* xref:../../windows_containers/enabling-windows-container-workloads.adoc#enabling-windows-container-workloads[Enabling Windows container workloads]
+* Understanding Windows container workloads
+* Enabling Windows container workloads
```

Comment:
Should you remove these two bullets as the referenced sections won't be available for searching? (I assume)